### PR TITLE
Small HTML Lesson 4 and 5 updates

### PR DESCRIPTION
**Pull request by mikestreety**

### CODE REVIEW

File: html/lesson5/tutorial.md [hunk 1]
```diff
@@ -158,8 +158,6 @@ Great! Now our images and links are on the left, and the main content on the rig
 
 [Border radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) enables us to create rounded corners for our elements. In the past, multiple divs and the use of images was required to achieve that effect.
 
-`border-radius: arc`
```
Comment (KimberleyCook) [inline html/lesson5/tutorial.md]: You are removing this example when above we explain what border radius is, can we keep it in please.

Comment (mikestreety) [inline html/lesson5/tutorial.md]: I was with a student last night and both me and the student agreed it seemed out of context - it also is not valid HTML. 

I could replace it with:

> Border radius can be defined in many different units - the higher the number, the bigger the curve. An example of border-radius in your code would be:

`border-radius: 10px;`

How does that sound?

Comment (KimberleyCook) [inline html/lesson5/tutorial.md]: Yeah that would be better, thank you :)

**Comment by mikestreety** [comment]
@KimberleyCook All fixed - glad I could help 👍

**Comment by KimberleyCook** [comment]
All merged, thank you very much for your contribution :)

**Pull request merged**